src/components/ImageUploader/index.js

```diff
@@ -12,8 +12,20 @@ import { StyledImageUploader, UploaderPreview, PreviewHover } from './styles';
 const getDefaultValuePreviews = files =>
   Array.isArray(files) ? files.map(file => file.preview) : [];
 
+const validateFilesType = (files, accept) => {
+  const formats = accept.split(',');
+  return files.reduce((isValid, file) => {
+    // if at least one file is not valid, return false
+    if (!isValid) return isValid;
+    return formats.includes(file.type);
```

Comment:
This may not work properly because:
1. `accept` may include spaces e.g. `accept="image/png, image/jpeg"`
2. `accept` may include wildcard e.g. `accept="image/*"`
3. `accept` may include extensions e.g. `accept=".png"`.

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#Unique_file_type_specifiers.